### DISCUSSION TITLE
Plans for `exampleSite` folder ?

### DISCUSSION THREAD

**Discussion by hugo-sid** [colinwilson/lotusdocs#3] [Feature Requests]
Hi @colinwilson . From the screenshot, lotus docs looks exiting.

Any plans to have the Hugo `exampleSite` folder ? So, that one can easily check out how to organise content.

**Comment by colinwilson** [comment]
> lotus docs looks exiting.

Thanks 🙏

Yes. An `exampleSite` directory is definitely a priority on the roadmap.

Content is currently organised [by weight](https://gohugo.io/templates/lists/#by-weight) in [Front Matter](https://gohugo.io/content-management/front-matter/#front-matter-variables).

The [repo](https://github.com/colinwilson/lotusdocs.dev) for the demo site, [lotusdocs.dev/docs/](https://lotusdocs.dev/docs/) has an early draft of how content is organised. I am also testing an option to configure the menu structure via `config.toml` as an alternative.

**Comment by colinwilson** [comment]
[`exampleSite`](https://github.com/colinwilson/lotusdocs/tree/release/exampleSite) directory completed.

**Reply by hugo-sid** [reply]
Thanks @colinwilson.
